cabot/metricsapp/api/grafana_elastic.py

```diff
@@ -40,6 +40,26 @@ def _get_terms_settings(agg):
     return terms_settings
 
 
+def _get_histogram_settings(agg):
+    """
+    Get the settings for a histogram aggregation
+    :param agg: the histogram aggregation json data
+    :return: dict of {setting_name: setting_value}
+    """
+    histogram_settings = dict(field=agg['field'])
+
+    settings = agg['settings']
+
+    interval = settings.get('interval')
```

Comment:
histograms have to have intervals right? so i don't think we have to have the check for whether "interval" exists